internal/ent/schema/validator.go

```diff
@@ -0,0 +1,24 @@
+package schema
+
+import (
+	"errors"
+	"fmt"
+	"net"
+)
+
+// IsValidIPAddress returns error if IP address is NOT valid
+func IsValidIPAddress(ip string) error {
```

Comment:
typically using `IsSomething` implies returning a `bool`. For things like this, I prefer to create a new package to isolate from the rest of the `schema` package.

Reply:
@rizzza updated, let me know if you like the package name or prefer different naming